src/litdata/streaming/item_loader.py

```diff
@@ -174,14 +177,34 @@ def load_item_from_chunk(
         if self._config.get("encryption"):
             data = self._load_encrypted_data(chunk_filepath, chunk_index, offset, encryption)
         else:
-            with open(chunk_filepath, "rb", 0) as fp:
-                data = self._load_data(fp, offset)
+            # with open(chunk_filepath, "rb", 0) as fp:
```

Comment:
Let's keep option for now. Also, let's use Imagenet for benchmarking ;)

Reply:
sure